samples/tests/playwright.config.ts

```diff
@@ -17,6 +17,8 @@ const config: PlaywrightTestConfig = {
       use: {
         browserName: 'chromium',
         permissions: ['notifications', 'camera', 'microphone'],
+        // Large enough to contain all UI elements from ComponentExamples sample
```

Comment:
fine to leave in for this PR, but in future PRs try to keep changes to solving one issue. So setting this viewport would ideally be its own PR tackling the issue of "Fix UI snapshot not capturing all component examples"